clusters/build-clusters/build05/ofcir/ofcir.yaml

```diff
@@ -0,0 +1,578 @@
+metadata:
+  name: ofcir-manager-config
+  namespace: ofcir-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: ofcir-controller-manager-metrics-service
+  namespace: ofcir-system
+spec:
+  ports:
+  - name: https
+    port: 8443
+    protocol: TCP
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: ofcir-service
+  namespace: ofcir-system
+spec:
+  ports:
+  - name: http
+    port: 8087
+    protocol: TCP
+  selector:
+    control-plane: controller-manager
+  type: NodePort
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: ofcir-controller-manager
+  namespace: ofcir-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: ofcir-api
+      labels:
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
+        name: kube-rbac-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 5m
+            memory: 64Mi
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        command:
+        - /ofcir-operator
+        image: registry.ci.openshift.org/ocp-kni/test:ofcir
+        imagePullPolicy: Always
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        name: ofcir-operator
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8081
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        securityContext:
+          allowPrivilegeEscalation: false
+      - command:
+        - /ofcir-api
+        image: registry.ci.openshift.org/ocp-kni/test:ofcir
+        imagePullPolicy: Always
+        name: ofcir-api
+        ports:
+        - containerPort: 8087
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 10m
+            memory: 64Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+      serviceAccountName: ofcir-controller-manager
+      terminationGracePeriodSeconds: 10
+---
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: ofcir-admin
+users:
+- afasano
+- dhiggins
+- tdomnesc
```

Comment:
The preferred way to do this is via Rover Groups: https://docs.ci.openshift.org/docs/how-tos/rbac/#rover-groups

Reply:
thanks, group created.